docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    t_1 = base[:50]
</pre><p> </p><p>Please use <code>T_A</code>  and <code>T_B</code>  to refer to time series A and time series B</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>